test-utils/src/lib.rs

```diff
@@ -179,14 +179,14 @@ pub async fn wait_n_finalized_blocks(n: usize, url: &str) {
 	use substrate_rpc_client::{ws_client, ChainApi};
 
 	let mut built_blocks = std::collections::HashSet::new();
-	let mut interval = tokio::time::interval(Duration::from_secs(3));
+	let mut interval = tokio::time::interval(Duration::from_secs(2));
```

Comment:
leaving a note on this line as a learning for future.

After speaking to @smohan-dw, realized that this change was very crucial.

If this interval is more than block-time, then we 'miss' the events triggered.  This has to be lesser than block time.  (In this case, less than 3seconds)

Reply:
Curious to know why `interval` is defined as mutable?